File: api-examples/rs/search/python/download.py
import requests

url = "https://<resource-server-ip>/resource-server/pscdcl/v1/download"

payload = "{\n\t\"resourceServerGroup\" : \"urn:iudx-catalogue-pune:pudx-resource-server/aqm-bosch-climo\",\n\t\"options\": \"all\"\n}"
headers = {
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data = payload)

print(response.text.encode('utf8'))

File: api-examples/rs/search/js/download.js
var settings = {
  "url": "https://<resource-server-ip>/resource-server/pscdcl/v1/download",
  "method": "POST",
  "timeout": 0,
  "headers": {
    "Content-Type": "application/json"
  },
  "data": "{\n\t\"resourceServerGroup\" : \"urn:iudx-catalogue-pune:pudx-resource-server/aqm-bosch-climo\",\n\t\"options\": \"all\"\n}",
};

$.ajax(settings).done(function (response) {
  console.log(response);
});
